guides/source/active_record_validations.md

```diff
@@ -590,7 +590,7 @@ These options are all supported:
 
 NOTE: The validator requires a compare option be supplied. Each option accepts a
 value, proc, or symbol. Any class that includes
-[Comparable](https://ruby-doc.org/3.3.5/Comparable.html) can be compared.
+[Comparable](https://ruby-doc.org/3.3.6/Comparable.html) can be compared.
```

Comment:
Let's link to official docs instead: https://docs.ruby-lang.org/en/3.3/Comparable.html
There are already a few occurences for that domain and it doesn't need to change for patch releases.

Could also do https://docs.ruby-lang.org/en/master/Comparable.html so it never has to change but on that I'm :shrug:

Reply:
would be good if there's something like https://docs.ruby-lang.org/en/latest/Comparable.html to point to latest release

Reply:
There is master: https://docs.ruby-lang.org/en/master/Comparable.html

Reply:
I have seen that across the guides most links point to the master branch of the official documentation (https://docs.ruby-lang.org/en/master/).

With that being said, I have modified all the occurrences of that domain to point to the `/master` documentation, and I have also replaced occurrences of un-official docs (like the original https://ruby-doc.org/3.3.6/Comparable.html), to point to the official documentation under master.

Reply:
@p8 I was looking explicitly for latest, not master, since there can be unreleased stuff.

Reply:
I've created an issue in the Ruby issue tracker to add "latest":
https://bugs.ruby-lang.org/issues/20887

Reply:
@p8 Thanks for opening that issue!

So, with that being said, what do you guys suggest to do with this PR? 
We either consider it for merging it or we wait for some feedback of https://bugs.ruby-lang.org/issues/20887 and a possible implementation of the `/latests` docs.